src/site/shared/tools/handlers/ToolHandler.ts

```diff
@@ -0,0 +1,24 @@
+import {Circuit}           from "core/public";
+import {InputManagerEvent} from "shared/utils/input/InputManagerEvent";
+
+
+export enum ToolHandlerResponse {
+    PASSTHROUGH = 0,
+    STOP = 1,
+}
+
+export interface ToolHandler {
```

Comment:
Maybe move to `DefaultTool.ts` so its more clear that's the exclusive user of this type, and to namespace it under `import {ToolHandler} from Default.ts`

Reply:
Will take care of this in https://github.com/OpenCircuits/OpenCircuits/pull/1248